configure.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Add custom sources"
+# Add ansible key
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367
+# Add custom sources
+cp $TEMPLATES_PATH/etc/apt/sources.list.d/* /etc/apt/sources.list.d/
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y ansible
+
+echo "Install web server tools"
+apt-get -qq install -y apache2
+chown -R www-data /var/www/
+
+echo "Set up base ansible"
+export ANSIBLE_ROLES_PATH=$ANSIBLE_ROLES_PATH:/home/vagrant/torque-sites/roles
+
+# Define env variables for ansible templates
+export DB_USERNAME=torque
+export DB_PASSWORD=torque
+export DEPLOYMENT_USER=$APP_USER
+export MEDIAWIKI_ADMIN_PASSWORD=admin_password
+export MEDIAWIKI_MWLIB_PASSWORD=mwlib_password
+export MEDIAWIKI_CSV2WIKI_PASSWORD=csv2wiki_password
```

Comment:
We don't have to solve this now, but the name "MEDIAWIKI_CSV2WIKI_PASSWORD" is a legacy name, right?  Are we still actually using `csv2wiki` in any part of the ETL process?

Reply:
Good question -- I just took everything that existed in an existing ansible script and included it here.  cc @frankduncan 

I think until we remove it from torque-sites we should keep it here, though.